### DISCUSSION TITLE
Missing entity from sun5kSG03LP1

### DISCUSSION THREAD

**Discussion by concierge-at** [kellerza/sunsynk#295] [Q&A] [answered]
Hello, thank you for providing the extension. Please excuse the English, it is not my native language. After much trial and error, I have now put together a small dashboard.

To finish it nicely I would add entities.

Load Power
PV 2 current
PV2 voltage

I have the whole system on the load so the load performance would be important. and I also have a microinverter on the gene port.
I think there is a separate entity for it.

Is it possible to use the generator input as + in the dashboard but have the balls run towards the inverter?

an please what is this non essential, i naver has seen. i only connect Grid, Load, and Microinverter und the Gen-MIC input
![dashboard-](https://github.com/kellerza/sunsynk/assets/135752342/8a6052e8-49b8-4821-a63d-671a1d9fb242)

Thanks in advance and apologies for the bumpy Google translation.
Thanks and all the best

**Comment by kellerza** [comment]
Non-essential will be loads connected between the grid CT clamp and the inverter. So this is loads connected to the grid and won't be backed up by solar/battery/gen, but can still get power from the inverter if the grid is stable and you configured the inverter to feed back into the grid/zero export.

The direction of the balls moving might need a template sensor to invert the direction, or maybe @slipx06 has another way to do this, you could ask here: https://github.com/slipx06/sunsynk-power-flow-card

**Comment by slipx06** [comment, marked as answer]
@concierge-at you can set `invert_aux: true` under the load section. You can also hide non_essential if you have everything connected to the inverter output. There is an option under the grid section, `show_nonessential: false`

**Reply by concierge-at** [reply]
-Thanky You Thank YOU!!
it works fine now:) with MQTT the world of data is open now, but for me ist it a book with 7 seals.

A little question to you is are there an register for day_aux (microinverterinput kwh) 
i have seeh the total pv Power is calculate in the sunsynk powerflow card. i am seeking a way to get the data vor the apex chart but this is an luxury problem on step 200 or more ^^

Thank you for this nice Integration, this wars the only way to work for me. i have test 2 version of ESP rs485, and 2 versions of USB-rs485 and no success. But this :) A+++ 
Is thera a way to safe the config wit the custom sensor wich i have type in manual?

have a sunny day
_conci
